### PR TITLE
[OTE_SDK_TESTS] Increase code coverage dice.py, config_element_type.py, label_mapper.py, visualization.py, prediction_to_annotation_converter.py

### PR DESCRIPTION
**Pull request by saltykox**
Test list:
•	ote_sdk/ote_sdk/tests/usecases/evaluation/test_dice.py
•	ote_sdk/ote_sdk/tests/configuration/enums/test_config_element_type.py
•	ote_sdk/ote_sdk/tests/serialization/test_label_mapper.py
•	ote_sdk/ote_sdk/tests/usecases/exportable_code/test_visualization.py: "show" and "is_quit" methods are uncovered due to requested "xcb" library
•	ote_sdk/ote_sdk/tests/usecases/exportable_code/test_prediction_to_annotation_converter.py

Task: CVS-74509

### CODE REVIEW

**Comment by saltykox** [comment]
run ote_sdk tests

**Comment by saltykox** [comment]
run ote_sdk tests

**Comment by saltykox** [comment]
run ote_sdk tests

**Comment by saltykox** [comment]
@LeonidBeynenson, @Ilya-Krylov please review

**Comment by Ilya-Krylov** [comment]
run ote_sdk tests

**Comment by Ilya-Krylov** [comment]
run ote_sdk tests

File: ote_sdk/ote_sdk/tests/serialization/test_label_mapper.py [hunk 1]
```diff
@@ -114,13 +124,13 @@ def test_flat_label_schema_serialization(self):
                 name=name,
                 domain=Domain.CLASSIFICATION,
                 creation_date=cur_date,
-                id=ID(i),
+                id=ID(str(i)),
```
Comment (Ilya-Krylov) [inline ote_sdk/ote_sdk/tests/serialization/test_label_mapper.py]: nit: str is not required

Comment (saltykox) [reply ote_sdk/ote_sdk/tests/serialization/test_label_mapper.py]: thanks, rolled back changes in test_flat_label_schema_serialization

**Comment by saltykox** [comment]
run ote_sdk tests

**Comment by saltykox** [comment]
run ote_sdk tests

**Pull request merged**